src/Dependencies/Collections/Internal/ThrowHelper.cs

```diff
@@ -279,7 +279,7 @@ private static string GetArgumentName(ExceptionArgument argument)
             }
         }
 
-        private static string GetResourceString(ExceptionResource resource)
+        public static string GetResourceString(this ExceptionResource resource)
```

Comment:
❗ This is not intended for use outside Microsoft.CodeAnalysis.Collections

Reply:
@sharwell What should I do instead?